src/main/java/core/basesyntax/utility/service/ReportService.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.utility.service;
+
+import core.basesyntax.utility.Pair;
+import java.util.List;
+
+public interface ReportService {
+    List<Pair<String, Integer>> createReport();
```

Comment:
For example this can be named as follows:
```suggestion
public interface ReportProvider {
    List<Pair<String, Integer>> provide();
```
because it has only 1 method without any parameters and returns some value. There are also useful words like `Consumer`, `Executor`, `Mapper`, and many others, so we always should keep in mind good naming